src/shelf_usage/zone.cc

```diff
@@ -219,12 +219,11 @@ the underlying Shelf Object. As the application/caller has already done
 the mmap, we just have to set the underlying shelf object fields
 appropriately and then initialize the Zone Header.
 */
-Zone::Zone(void *addr,
-           size_t max_pool_size, void *helper, size_t helper_size) :
-    shelf_location_ptr((char*)addr),
-    header_ptr((char*)helper)
-{
+Zone::Zone(void *addr, size_t max_pool_size, void *helper, size_t helper_size,
```

Comment:
Can you change copyright in all the files modified as part of this PR to include 2021. eg heap.h copyright should be "2016-2021"